src/components/Header/SubComponents/CartModal/CartModal.tsx

```diff
@@ -45,9 +25,14 @@ export default function CartModal() {
   const handleClickOutside = () => {
     setShowCart(false)
   }
+
+  const handleRemoveAll = () => {
+    handleRemoveAllItems()
+  }
+
   useClickOutside([cartCardRef, cartButtonRef], handleClickOutside)
   const renderEmptyCart = (
-    <div className={Style['cart-modal__container__body']}>
+    <div className={Style['cart-modal__container__body-empty']}>
```

Comment:
what do you think after adding to cart it would show the modal?

Reply:
Show the cart modal? Alright let me try that.